core/src/avm2/globals/flash/desktop/Clipboard.as

```diff
@@ -0,0 +1,52 @@
+package flash.desktop {
+    public class Clipboard {
```

Comment:
This should import `__ruffle__.stub_method`/`getter` up here, and imports (the stub imports and the `flash.system.System` import) should be placed between the `package` block and the `public class` block